Hello, here is the 13icons you asked on twitter

Virgile-fr opened the pull request:
Hello Gil,

I hope this message finds you well. Following [your tweet](https://twitter.com/gilgNYC/status/1753165604695994686) seeking a designer for shipping carrier icons that match Active Merchant's payment icons, I'm excited to let you know that I've completed the design work and believe I've created icons that fit the bill.

Hope it's working for you :)

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Update svg_Mondial_relay.svg

Virgile-fr committed:
Update svg_Poste_italiane.svg

Virgile-fr committed:
Update and rename svg_Mondial_relay.svg to mondialrelay.svg

Virgile-fr committed:
Update and rename svg_Poste_italiane.svg to posteitaliane.svg

Virgile-fr committed:
Update posteitaliane.svg

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Update chronopost.svg

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Update evri.svg

Virgile-fr committed:
Update sendle.svg

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Update ups.svg

Virgile-fr committed:
Update ups.svg

Virgile-fr committed:
Add files via upload

Virgile-fr committed:
Add files via upload

source file with the 13 icons

Virgile-fr committed:
optimize icons with gvgo, delete old icons

Virgile-fr committed:
update yml

Virgile-fr committed:
change svg folder location

Virgile-fr committed:
Delete svg/.DS_Store

Virgile-fr committed:
Delete .DS_Store

Virgile-fr committed:
Delete db/.DS_Store

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
+  label: Evri
+  group: other
+-
+  name: dpd
```
For DPD, I think it has to be horizontal per guidelines: https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf
![CleanShot 2024-02-02 at 12 23 15@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/89a0b5a7-6ec5-4a0c-ba49-17ee49ab47cb)

![CleanShot 2024-02-02 at 12 23 35@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/fde7a30b-503f-416d-a01e-ebbd2ee3cf3b)

Any thoughts on providing both the icon and then one with the symbol+logo. maybe dpd_symbol.svg
![CleanShot 2024-02-02 at 12 26 13@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/3b44517c-f153-4b41-a535-1237af8d263a)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
+  label: Colissimo
+  group: other
+-
+  name: chronopost
```
![CleanShot 2024-02-02 at 12 26 49@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/8021eb98-4ccf-4e23-8b75-4a164f83f68d)

Like DPD, we can provide a chronopost_symbol.svg with just the chronopost blue box. https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
   name: ups
```
Per UPS brand guidelines, can we place the logo on white background? 

![CleanShot 2024-02-02 at 12 30 11@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/a9b0362d-cf6b-4759-a538-78fccaa54c2a)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
```
Can you supply a usps_symbol.svg variant? Just the symbol without "USPS" text

![CleanShot 2024-02-02 at 12 29 13@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4adff7a5-3ede-4c53-a9c0-a65468b82216)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
+  label: Correos
+  group: other
+-
+  name: colissimo
```
Could you create a colissimo_symbol.svg version: https://www.colissimo.entreprise.laposte.fr/en/tools-and-services/communication-kit

![CleanShot 2024-02-02 at 12 35 35@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/9a4da96b-059a-4be0-b637-24e6c8b12dc0)

gil-- replied:
![CleanShot 2024-02-02 at 12 39 54@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4ae3d5d0-a1c6-4892-b388-5d937dfbeef4)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
 -
   name: fedex
-  label: FedEx
+  label: Fedex
```
Can we remove "Express" and just have FedEx logo? Adding Express can limit uses when it mig
![CleanShot 2024-02-02 at 12 39 15@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/c08ead23-661f-4460-b6c4-6d5bd5f87838)
ht be ground/overnight/freight.

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
```
Can we make sure to use the red logo on white background per https://www.mondialrelay.fr/solutionspro/documentation-technique/kit-de-communication/

![CleanShot 2024-02-02 at 12 41 00@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/10f3ec05-b757-4a5a-8c56-8d7687989711)

If you're up for it, include a mondial-relay_symbol.svg variant of just the symbol:


![CleanShot 2024-02-02 at 12 42 04@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/3ec4888d-79f3-4f1d-898f-deba34551408)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
   label: DHL
   group: other
+-
+  name: correos
```
Per Correos new image guidelines, can you only include the symbol?

https://www.correos.com/en/sala-prensa/#imagen-marca

Alternatively, we could just add a correos_symbol.svg variant?

![CleanShot 2024-02-02 at 12 43 53@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4670f3e1-5edd-4ade-9e57-1b415d6c82fa)

gil-- commented on db/carrier_icons.yml:
```diff
@@ -2,11 +2,52 @@
+  label: Chronopost
+  group: other
+-
+  name: canadapost
```
Would you be able to add a canadapost_symbol.svg variant?

https://www.canadapost-postescanada.ca/cpc/en/our-company/news-and-media/media-centre/canada-post-logos.page

![CleanShot 2024-02-02 at 12 47 00@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/92d7af2d-511d-4e8c-b370-0111118d35fc)

gil-- reviewed (changes requested):
Thank you @Virgile-fr so much for helping with this! Have some requested changes to ensure we more closely match official brand guidelines.

Send a DM to https://twitter.com/gilgNYC with your Paypal email and after merging, I'll send $20/icon to your Paypal.

Virgile-fr commented:
Hi  @gil--  ,

Thank you for your feedback. No problem with the changes; I'm on it. I'll update you as soon as possible.

Virgile-fr committed:
adding:   svg/canadapost_symbol.svg
        svg/chronopost_symbol.svg
        svg/colissimo_symbol.svg
        svg/correos_symbol.svg
        svg/dpd_symbol.svg
        svg/mondialrelay_symbol.svg
        svg/usps_symbol.svg

Virgile-fr commented:
Hey @gil--  , Just to keep you updated, all the requested changes have been made. I've included the variants, corrected the logos, and the YML file is up to date.

There are now 20 variations in total, considering the additions 

I hope this matches what you were looking for. Once you've had a chance to check everything, I'll be ready to share my PayPal details through Twitter DM if all is well.

Best wishes.

gil-- reviewed (approved):
Thanks for implementing the changes and the symbols!
I think this is a great v1 and will hopefully be useful for the wider community.

gil-- commented:
@Virgile-fr Send me a DM on Twitter when you have time.

Virgile-fr commented:
Hi @gil--  , really happy that everything is working :) , I sent you a DM on @gilgNYC from my Twitter account (@irgile), but I think the messages are being marked as spam... I'll send you another DM from another account (@sidfik). Let me know if you receive it properly. :)

If not, try to send me a message at this link: https://twitter.com/irgile. There should be a request for messages. :)
